mod/friends_collections/views/default/collections/collection.php

```diff
@@ -57,16 +57,15 @@
 
 $params = [
 	'collection' => $collection,
-	'metadata' => $menu,
+	'metadata' => false,
```

Comment:
This is not the correct way to fix this issue.
`$menu` is still used in the listing view of a collection and still needs to be displayed.

In the `if` statement which checks `$full_view` `$metadata` needs to be set to `false` (around line 41)